packages/mattermost-components/src/generic_modal/generic_modal.tsx

```diff
@@ -0,0 +1,521 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React from 'react';
+import {Modal} from 'react-bootstrap';
```

Comment:
@crspeller @hmhealey Except for animation are we gaining anything from `react-bootstrap`, I was going through this example https://medium.com/@oherterich/from-react-transition-group-to-framer-motion-fab25f1e55a9#:~:text=Framer%20Motion%20is%20a%20very,or%20layout%20%E2%80%94%20all%20in%20Javascript. of React transition group vs framer motion.